dom/src/test/java/org/apache/james/mime4j/message/MessageBuilderTest.java

```diff
@@ -46,7 +47,7 @@
 
     @Test
     public void testSetCustomBody() throws Exception {
-        MessageBuilder builder = MessageBuilder.create();
+        MessageBuilder builder = MessageBuilder.of();
```

Comment:
the setCustomBody test was creating the builder with create and now it uses of. As the way you create the builder can make a difference, you should have the test duplicated for each builder.
Actually, I think it shows that we now have two different builders with the same name.